### PR TITLE
doc: sync doc about enable auth for docker image

### PR DESCRIPTION
**Pull request by aroundabout**
## Purpose of the PR

> I will sync the en version later

1. sync with [2403](https://github.com/apache/incubator-hugegraph/pull/2403)
2. change the order of the changelogs (there is a small problem in the origin order), and make the weight larger so that we can set the weight for version 1.5.0 to 10-1=9, where 10 is the weight for 1.2.0
3. Put the relevant parts of docker first

## Main changes 

as said in Purpose

### CODE REVIEW

File: content/cn/docs/config/config-authentication.md [hunk 1]
```diff
@@ -73,13 +73,15 @@ gremlin.graph=org.apache.hugegraph.auth.HugeFactoryAuthProxy
 
 在鉴权配置完成后，需在首次执行 `init-store.sh` 时命令行中输入 `admin` 密码 (非 docker 部署模式下)
 
-如果基于 docker 镜像部署或者已经初始化 HugeGraph 并需要转换为鉴权模式，需要删除相关图数据并重新启动 HugeGraph，若图已有业务数据，暂时**无法直接转换**鉴权模式 (对于该功能的改进将在下个版本发布，修改方式可参考 [PR 2411](https://github.com/apache/incubator-hugegraph/pull/2411))。
+如果基于 docker 镜像部署或者已经初始化 HugeGraph 并需要转换为鉴权模式，需要删除相关图数据并重新启动 HugeGraph, 若图已有业务数据，暂时**无法直接转换**鉴权模式 (hugegraph 版本 <= 1.2.0) 
+> 对于该功能的改进已经在最新版本发布，修改方式可参考 [PR 2411](https://github.com/apache/incubator-hugegraph/pull/2411), 此时无需删除原本存储的数据。
```
Comment (imbajin) [inline content/cn/docs/config/config-authentication.md:77]:
```suggestion
> 对于该功能的改进已经在最新版本发布(Docker latest 可用)，可参考 [PR 2411](https://github.com/apache/incubator-hugegraph/pull/2411), 此时可无缝切换。 
```

File: content/cn/docs/quickstart/hugegraph-hubble.md [hunk 2]
```diff
@@ -36,11 +36,45 @@ HugeGraph 是一款面向分析型，支持批量操作的图数据库系统，
...
+或者使用 docker-compose 启动 hubble，另外如果 hubble 和 graph 在同一个 docker 网络下，可以使用 graph 的 contain_name 进行访问，而不需要宿主机的 ip
+
+使用`docker-compose up -d`，`docker-compose.yml`如下：
+
+```yaml
+version: '3'
+services:
+  server:
+    image: hugegraph/hugegraph
+    container_name: graph
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-hubble.md:60]:
```suggestion
    container_name: server
```

server 是不是好一点? 不过要注意统一修改

File: content/cn/docs/quickstart/hugegraph-hubble.md [hunk 3]
```diff
@@ -36,11 +36,45 @@ HugeGraph 是一款面向分析型，支持批量操作的图数据库系统，
...
+  hubble:
+    image: hugegraph/hubble
+    container_name: hubble
+    ports:
+      - 8088:8088
+```
+
+> 注意：
+>
+> 1. hugegraph-hubble 的 docker 镜像是一个便捷版本，用于快速启动 hubble，并不是**官方发布物料包方式**。你可以从 [ASF Release Distribution Policy](https://infra.apache.org/release-distribution.html#dockerhub) 中得到更多细节。
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-hubble.md:73]:
```suggestion
> 1. `hugegraph-hubble` 的 docker 镜像是一个便捷发布版本，用于快速测试试用 hubble，并非**ASF官方发布物料包的方式**。你可以从 [ASF Release Distribution Policy](https://infra.apache.org/release-distribution.html#dockerhub) 中得到更多细节。
```

File: content/cn/docs/quickstart/hugegraph-hubble.md [hunk 4]
```diff
@@ -36,11 +36,45 @@ HugeGraph 是一款面向分析型，支持批量操作的图数据库系统，
...
+>
+> 2. 推荐使用 `release tag`(如 `1.2.0`) 以获取稳定版。使用 `latest` tag 可以使用开发中的最新功能。
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-hubble.md:75]:
```suggestion
> 2. **生产环境**推荐使用 `release tag`(如 `1.2.0`) 稳定版。使用 `latest` tag 默认对应 master 最新代码。
```

File: content/cn/docs/quickstart/hugegraph-loader.md [hunk 5]
```diff
@@ -103,7 +66,45 @@ services:
...
+
+#### 2.3 克隆源码编译安装
+
+克隆最新版本的 HugeGraph-Loader 源码包：
+
+```bash
+# 1. get from github
+git clone https://github.com/apache/hugegraph-toolchain.git
+
+# 2. get from direct  (e.g. here is 1.0.0, please choose the latest version)
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-loader.md:88]:
```suggestion
# 2. get from direct url (please choose the **latest release** version)
```

File: content/cn/docs/quickstart/hugegraph-loader.md [hunk 6]
```diff
@@ -103,7 +66,45 @@ services:
...
+
+由于 Oracle ojdbc license 的限制，需要手动安装 ojdbc 到本地 maven 仓库。
+访问[Oracle jdbc 下载](https://www.oracle.com/database/technologies/appdev/jdbc-drivers-archive.html) 页面。选择 Oracle Database 12c Release 2 (12.2.0.1) drivers，如下图所示。
+
+打开链接后，选择“ojdbc8.jar”
+
+把 ojdbc8 安装到本地 maven 仓库，进入`ojdbc8.jar`所在目录，执行以下命令。
+```
+mvn install:install-file -Dfile=./ojdbc8.jar -DgroupId=com.oracle -DartifactId=ojdbc8 -Dversion=12.2.0.1 -Dpackaging=jar
+```
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-loader.md:100]: 考虑折叠 ojdbc 的内容, 用户已经很少了 (后续不影响编译的话可以进一步压缩)

File: content/cn/docs/config/config-computer.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph-Computer 配置"
-linkTitle: "Config Computer"
+linkTitle: "HugeGraph-Computer 配置"
```
Comment (imbajin) [inline content/cn/docs/config/config-computer.md:3]:
```suggestion
linkTitle: "图计算 Computer 配置"
```

File: content/cn/docs/config/config-guide.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph 配置"
-linkTitle: "Config Guide"
+linkTitle: "配置指南"
```
Comment (imbajin) [inline content/cn/docs/config/config-guide.md:3]:
```suggestion
linkTitle: "参数配置"
```

File: content/cn/docs/config/config-https.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "配置 HugeGraphServer 使用 https 协议"
-linkTitle: "Config Https"
+linkTitle: "配置 HugeGraphServer 使用 https 协议"
```
Comment (imbajin) [inline content/cn/docs/config/config-https.md:3]:
```suggestion
linkTitle: "配置 HTTPS"
```

File: content/cn/docs/config/config-option.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph 配置项"
-linkTitle: "Config Options"
+linkTitle: "HugeGraph 配置项"
```
Comment (imbajin) [inline content/cn/docs/config/config-option.md:3]:
```suggestion
linkTitle: "配置项列表"
```

File: content/cn/docs/guides/architectural.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph Architecture Overview"
-linkTitle: "Architecture Overview"
+linkTitle: "架构总览"
```
Comment (imbajin) [inline content/cn/docs/guides/architectural.md:3]:
```suggestion
linkTitle: "架构概览"
```

File: content/cn/docs/guides/backup-restore.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Backup Restore"
-linkTitle: "Backup Restore"
+linkTitle: "备份与恢复"
```
Comment (imbajin) [inline content/cn/docs/guides/backup-restore.md:3]:
```suggestion
linkTitle: "备份 & 恢复"
```

File: content/cn/docs/guides/desgin-concept.md [hunk 13]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph Design Concepts"
-linkTitle: "Design Concepts"
+linkTitle: "设计理念"
```
Comment (imbajin) [inline content/cn/docs/guides/desgin-concept.md:3]:
```suggestion
linkTitle: "设计理念(legacy)"
```

File: content/cn/docs/performance/hugegraph-benchmark-0.5.6.md [hunk 14]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph BenchMark Performance"
-linkTitle: "HugeGraph BenchMark Performance"
+linkTitle: "HugeGraph 基准性能"
```
Comment (imbajin) [inline content/cn/docs/performance/hugegraph-benchmark-0.5.6.md:3]:
```suggestion
linkTitle: "HugeGraph 基准测试"
```

File: content/cn/docs/quickstart/hugegraph-client.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph-Client Quick Start"
-linkTitle: "Develop with HugeGraph-Client"
+linkTitle: "使用 HugeGraph-Client 进行开发"
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-client.md:3]:
```suggestion
linkTitle: "使用 HugeGraph-Client 开发"
```

Comment (imbajin) [reply content/cn/docs/quickstart/hugegraph-client.md:3]: 另外在 client 的页面里加一下链接一下 Go-Client 的 README 地址 (中/英两个版本)

File: content/cn/docs/quickstart/hugegraph-computer.md [hunk 16]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph-Computer Quick Start"
-linkTitle: "Analysis with HugeGraph-Computer"
+linkTitle: "使用 HugeGraph-Computer 进行数据分析"
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-computer.md:3]:
```suggestion
linkTitle: "使用 Computer 进行 OLAP 分析"
```

File: content/cn/docs/quickstart/hugegraph-hubble.md [hunk 17]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph-Hubble Quick Start"
-linkTitle: "Visual with HugeGraph-Hubble"
+linkTitle: "使用 HugeGraph-Hubble 可视化"
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-hubble.md:3]:
```suggestion
linkTitle: "使用 Hubble 实现图可视化"
```

File: content/cn/docs/quickstart/hugegraph-loader.md [hunk 18]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph-Loader Quick Start"
-linkTitle: "Load data with HugeGraph-Loader"
+linkTitle: "使用 HugeGraph-Loader 导入数据"
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-loader.md:3]:
```suggestion
linkTitle: "使用 Loader/Spark 实时导入数据"
```

File: content/cn/docs/quickstart/hugegraph-loader.md [hunk 19]
```diff
@@ -103,7 +66,50 @@ services:
 > 
 > 1. hugegraph-loader 的 docker 镜像是一个便捷版本，用于快速启动 loader，并不是**官方发布物料包方式**。你可以从 [ASF Release Distribution Policy](https://infra.apache.org/release-distribution.html#dockerhub) 中得到更多细节。
 > 
-> 2. 推荐使用 `release tag`(如 `1.0.0`) 以获取稳定版。使用 `latest` tag 可以使用开发中的最新功能。
+> 2. 推荐使用 `release tag`(如 `1.2.0`) 以获取稳定版。使用 `latest` tag 可以使用开发中的最新功能。
+
+#### 2.2 下载已编译的压缩包
+
+下载最新版本的 HugeGraph-Toolchain Release 包，里面包含了 loader + tool + hubble 全套工具，如果你已经下载，可跳过重复步骤
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-loader.md:73]:
```suggestion
下载最新版本的 `HugeGraph-Toolchain` Release 包，里面包含了 `loader + tool + hubble` 全套工具，如果你已经下载，可跳过重复步骤
```

File: content/cn/docs/quickstart/hugegraph-tools.md [hunk 20]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "HugeGraph-Tools Quick Start"
-linkTitle: "Manage with HugeGraph-Tools"
+linkTitle: "使用 HugeGraph-Tools 进行管理"
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-tools.md:3]:
```suggestion
linkTitle: "使用 HugeGraph-Tools 导出/管理图"
```

File: content/cn/docs/changelog/hugegraph-0.12.0-release-notes.md [hunk 21]
```diff
@@ -1,14 +1,15 @@
 ---
 title: "HugeGraph 0.12 Release Notes"
 linkTitle: "Release-0.12.0"
-weight: 2
+draft: true
```
Comment (aroundabout) [inline content/cn/docs/changelog/hugegraph-0.12.0-release-notes.md:4]:
```suggestion
```

**Pull request merged**